src/pages/cluster-management/cluster-configuration.html

```diff
@@ -4,6 +4,13 @@
     <h1>{{pageTitle}}</h1>
     <div onto-loader-new ng-if="loader" class="cluster-loader" message="getLoaderMessage" size="75"></div>
 
+    <div ng-if="!loader && noClusterExists">
+        <p class="alert alert-danger" ng-show="noClusterExists">
```

Comment:
Double check for noClusterExists. Do you need both?

Reply:
Not any more. Removed it.